profile/pathAggregator.go

```diff
@@ -0,0 +1,341 @@
+// This code is directly taken from github.com/accuknox/auto-policy-discovery/src/common
```

Comment:
can we import this package instead of rewriting it? Is it modified according to karmor probe?

Reply:
The reason is: The accunox packages are dependent on cilium but the part of code which we require is independent of cilium. Using the package will introduce cilium dependency which will make profiling unavailable for windows as it can't be compiled on windows!